centre-web/src/components/AuthManagement/EditAccess/index.tsx

```diff
@@ -190,14 +228,33 @@ export const EditAccessModal = ({ app }: EditAccessModalProps) => {
             </Grid>
           )}
 
+          {/* Please Note section for apps with granular role management */}
+          <If condition={supportsGranularRoleManagement && selectedRole && selectedRole !== "revoke" && selectedRole !== "deny"}>
```

Comment:
Let's replace the magic strings with references to this object:

<img width="510" height="206" alt="Image" src="https://github.com/user-attachments/assets/0d9a8f00-2ae0-48ab-8a26-d29ef962846a" />